Assets/Plants/BranchMotor.cs

```diff
@@ -86,6 +86,18 @@ public void CreateNewChildBranch(Vector3 direction, KeyValuePair<Vector3, Vector
         Instantiate(branch, positionAndNormal.Key, quat, transform.parent);
     }
 
+    /// <summary>
+    /// Create a branch in the continuity of
+    /// the given position and normal.
+    /// </summary>
+    /// <param name="positionAndNormal">Position (Vector3) and Normal (Vector3).</param>
+    public void CreateNewChildBranchContinuity(KeyValuePair<Vector3, Vector3> positionAndNormal)
+    {
+        Quaternion quat = Quaternion.FromToRotation(Vector3.up, positionAndNormal.Value);
```

Comment:
@LibertAntoine
Done like this because the other method did not work.
Maybe we should review the way the quaternion is computed in CreateNewChildBranch method?